src/modules/expression/binop/range.rs

```diff
@@ -59,7 +60,10 @@ impl TranslateModule for Range {
         let from = self.from.translate(meta);
         let to = self.to.translate(meta);
         if self.neq {
-            let to_neq = translate_computation(meta, ArithOp::Sub, Some(to), Some("1".to_string()));
+            let to_neq = match &self.to.value.as_ref().unwrap() {
+                ExprType::Number(_) => (to.parse::<usize>().unwrap() - 1).to_string(),
```

Comment:
Please check your edge cases:
```
hwalters@Ghostwheel ~/git/amber (range-neq-opt) 
$ cat range.ab
#!/usr/bin/env amber
loop num in 0..0 {
    echo num
}
```
```
hwalters@Ghostwheel ~/git/amber (range-neq-opt) 
$ amber range.ab -
#!/usr/bin/env bash
# Written in [Amber](https://amber-lang.com/)
# version: 0.3.4-alpha
# date: 2024-10-15 21:47:28
# #!/usr/bin/env amber
for num in $(seq 0 18446744073709551615); do
    echo ${num}
done
```

Reply:
Also need to write a unit test to check this!

Reply:
Looks good otherwise.

Reply:
I'm also unsure why you needed to change signed `to.parse::<i64>()` to unsigned `to.parse::<usize>()`. Looks like Bash `seq` command is entirely capable of handling negative numbers, but your latest uses the old mechanism:
```
hwalters@Ghostwheel ~/git/amber (range-neq-opt) 
$ cat range.ab 
#!/usr/bin/env amber
loop num in -3..-1 {
    echo num
}
```
```
hwalters@Ghostwheel ~/git/amber (range-neq-opt) 
$ amber range.ab -
#!/usr/bin/env bash
# Written in [Amber](https://amber-lang.com/)
# version: 0.3.4-alpha
# date: 2024-10-15 21:52:26
# #!/usr/bin/env amber
for num in $(seq $(echo '-' 3 | bc -l | sed '/\./ s/\.\{0,1\}0\{1,\}$//') $(echo $(echo '-' 1 | bc -l | sed '/\./ s/\.\{0,1\}0\{1,\}$//') '-' 1 | bc -l | sed '/\./ s/\.\{0,1\}0\{1,\}$//')); do
    echo ${num}
done
```
In fact, do you need `translate_computation()` at all?

Reply:
theres also `isize`

Reply:
Changed to use `isize`, which fixes the `0..0` edge case.